client/js/is-sum-formula.js

```diff
@@ -0,0 +1,40 @@
+const isSumFormula = function(string, numCols, numRows) {
+	// return [col, startRow, endRow] if string is a sum function
+	// otherwise return false
+
+	// string must be in form
+	// =SUM([One capital letter][1 or more digits]:[One capital letter][1 or more digits])
+	const re = /=SUM\(([A-Z]\d+):([A-Z]\d+)\)$/;
+
+	// if string is in valid format, get col letters and row numbers
+	if (string.match(re) !== null) {
+
+		const numberPattern = /\d+/g;
```

Comment:
Indent with spaces not tabs please.

Look at how huge the indent looks, with tabs:
![image](https://cloud.githubusercontent.com/assets/17621133/25734174/45428406-3115-11e7-9cd6-6c16c570bbcc.png)